.travis.yml

```diff
@@ -16,6 +16,7 @@ android:
 
 before_install:
   - pip install --user codecov
+  - brew install infer
```

Comment:
Oh CircleCI is running on linux? Anyways so far it does not matter since the gradle plugin fails and this one needs to be fixed first.